Helper/Vault.php

```diff
@@ -233,4 +235,14 @@ private function getExtensionAttributes(InfoInterface $payment)
         }
         return $extensionAttributes;
     }
+
+    /**
+     * @param $additionalData
+     * @return bool
+     */
+    private function isGooglePay($additionalData): bool
```

Comment:
What about moving this method to `\Adyen\Payment\Helper\PaymentMethods` so that we can reuse it?

Reply:
For now it is not used from other class and `\Adyen\Payment\Helper\PaymentMethods` I think is be responsible for  fetching the payment methods and functionality around it

Reply:
I saw that you also added an `isGooglePay` method in `Model/Billing/Agreement.php`.

Reply:
Check again ?

Reply:
Looks good to me! 👍